spec/index.html

```diff
@@ -1492,12 +1511,12 @@ <h2>Security Considerations</h2>
 
 <section id="internationalization" class="appendix informative">
   <h2>Internationalization Considerations</h2>
-  <p>RDF is restricted to representing string values with left-to-right or right-to-left direction indicators.
+  <p>RDF is restricted to representing Unicode <a>string</a> [[UNICODE]] values with left-to-right or right-to-left direction indicators.
```

Comment:
This sentence calls for more attention for reasons other than its use of `string` as a term. I think it is factually inaccurate or maybe some words were left out. Perhaps:

```suggestion
<p>RDF does not provide a means of associating <a>base direction</a> with a <a>string</a> other than via the inclusion of Unicode [[UNICODE]] bidirectional control characters inside the string itself.
```

Reply:
Note that there is a long-pending PR #48 on adding base direction, which I believe touches this. That PR has been held up for a long time due to lack of consensus and understanding on the requirements for addressing text direction in RDF. I believe this needs to be resolved on a call with interested parties and hope that time at TPAC can be reserved to address this.

Reply:
Totally. I'll send an email to the various chairs to see if we can align schedules.